examples/custom_vocabularies_example.py

```diff
@@ -0,0 +1,52 @@
+"""Copyright 2019 REV
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import time
+from rev_ai import customvocabulariesclient
+from rev_ai.models import CustomVocabulary
+
+access_token = "your_access_token"
+
+print("Creating Custom Vocab Client")
+client = customvocabulariesclient.RevAiCustomVocabulariesClient(access_token)
+
+print("Submitting")
+custom_vocabularies_job = client.submit_custom_vocabularies(
+    [
+        {"phrases": ["Peace out"]},
```

Comment:
I think we should use more professional terms in our example. Let's use the names of well known members of the tech community that have unique names.

Reply:
In our documentation for the Async API we use this example:
```
"custom_vocabularies": [
{
"phrases": [
"Paul McCartney",
"Amelia Earhart"
]
}
],
```